linera-chain/src/manager.rs

```diff
@@ -358,34 +358,40 @@ where
         Ok(Outcome::Accept)
     }
 
-    /// Checks if the current round has timed out, and signs a `Timeout`.
+    /// Checks if the current round has timed out, and signs a `Timeout`. Returns `true` if the
+    /// chain manager's state has changed.
```

Comment:
I find the name of this function to be misleading.
With the existing API, the name should be `has_manager_state_changed` with the error pattern being whether it has timeout or not. But the error pattern is usually something that is in the background and not in the name of the function.

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/4327/commits/22de1e47995b6b0dd11b2a9d5c5b2680a807d1d0.